src/components/EventForm/index.tsx

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
+import CoreTypography from "src/components/core/typography";
+import colors from "src/components/core/colors";
+import Container from "@material-ui/core/Container";
+import Link from "@material-ui/core/Link";
+import constants from "utils/constants";
+import { Checkbox, FormControlLabel } from "@material-ui/core";
+import DescriptionIcon from "@material-ui/icons/Description";
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        container: {
+            display: "flex",
+            textAlign: "center",
+            width: "750px",
+            borderStyle: "solid",
+            borderWidth: "2px",
+            borderColor: colors.green,
+            height: "700px",
+            paddingLeft: "0",
+        },
+
+        leftWrapper: {
+            backgroundColor: colors.green,
+            color: colors.white,
+            paddingTop: "220px",
+        },
+
+        rightWrapper: {
+            paddingTop: "50px",
+            width: "310px",
+        },
+
+        textWrapper: {
+            paddingBottom: "20px",
+        },
+
+        form: {
+            textAlign: "left",
+        },
+
+        input: {
+            backgroundColor: colors.lightGray,
+            border: "none",
+            height: "40px",
+            fontSize: "20px",
+        },
+
+        inputLabel: {
+            paddingTop: "25px",
+            paddingBottom: "5px",
+        },
+
+        bottomContainer: {
+            paddingLeft: "0px",
+            paddingTop: "5px",
+        },
+
+        waiverLink: {
+            color: colors.grays[80],
+        },
+
+        waiverWrapper: {
+            paddingLeft: "0px",
+        },
+
+        checkboxText: {
+            width: "240px",
+            paddingTop: "10px",
+        },
+
+        button: {
+            marginTop: "25px",
+            marginLeft: "140px",
+            padding: "15px",
+            backgroundColor: colors.green,
+            border: "none",
+            color: colors.white,
+            borderRadius: "6px",
+            fontSize: "20px",
+            fontFamily: "Roboto",
+        },
+    })
+);
+
+export default function EventsForm() {
```

Comment:
Can you name this `EventForm` to be consistent with the filename? I actually like `EventSignUp` better.